src/html/index.html

```diff
@@ -78,12 +81,14 @@ <h1>Overzicht gemaakte query's</h1>
             pellentesque at risus. Nunc ex elit, tincidunt
             at lorem vel, mattis gravida nisl. Praesent sodales nibh massa, id
             ultrices metus viverra id.
-          </div>
-          <input type="button" class="btn btn--small btn--inline" value="Edit">
+          </p>
+
+          <input id="redirect" type="button" class="btn btn--small btn--inline" value="Edit">
```

Comment:
Deze knop gaat toch weg hé, maar:

Input type="button" is bedoeld om formulieren te verzenden.
Default gaat dit bv uw pagina refreshen.

Gebruik `<button></button>` voor knoppen waar je zelf functionaliteit op zet.

Reply:
Done!